src/EnergyPlus/SingleDuct.cc

```diff
@@ -3066,13 +3089,43 @@ void SingleDuctAirTerminal::SizeSys(EnergyPlusData &state)
                 MinAirFlowFracDes = 0.0;
             }
         }
+        if (SizingDesRunThisAirSys) {
+            if (state.dataSize->SysSizInput(SysSizNum).SystemOAMethod == SOAM_SP) { // 62.1 simplified procedure
+                if (this->MaxAirVolFlowRate > 0.0) {
+                    MinAirFlowFracDes = 1.5 *
+                                        max(state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).VozClgByZone,
+                                            state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).VozHtgByZone) /
+                                        this->MaxAirVolFlowRate;
+
+                    // adjust maximum flow rate
+                    if (MinAirFlowFracDes > 1.0 && IsMaxFlowAutoSize) {
+                        this->MaxAirVolFlowRate *= MinAirFlowFracDes;
```

Comment:
This increasing of max terminal airflow rate seems a bit unexpected.  I wonder if a message to the user that this happened would be useful.  I can see users wondering why the size of the air handler goes up with the method.

Reply:
Sure, what do you think of the following:
```
ShowWarningError(state, "SingleDuctSystem:SizeSys: Autosized maximum air flow rate for " + this->SysName + " was increased to meet the zone primary air flow determined according the ASHRAE Standard 62.1 Simplified Procedure.");
```
This could be used for in SingleDuct.cc and PoweredInductionUnits.cc.